packages/authentication/src/metadata.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as assert from 'assert';
+import 'reflect-metadata';
+
+export function authenticate(controllerClass: Object, methodName: string) {
+  Reflect.defineMetadata('loopback:authenticate', true, controllerClass, methodName);
+}
+
+export function getAuthenticateMetadata(controllerObj: Object, methodName: string): boolean {
+  return Reflect.getMetadata('loopback:authenticate', controllerObj, methodName) || 'none';
```

Comment:
Is it a good idea to return `none`? Wouldn't it better to return `undefined` instead?